test/test.rs

```diff
@@ -41,12 +41,10 @@ macro_rules! skip_if_jailed {
     ($name:expr) => {
         use ::sysctl::CtlValue;
 
-        if let CtlValue::Int(1) = ::sysctl::value("security.jail.jailed")
-            .unwrap()
-        {
+        if let CtlValue::Int(1) = ::sysctl::value("security.jail.jailed").unwrap() {
```

Comment:
Please don't mix formatting changes with content changes.

Reply:
I see. I will revert the commit that changes the style and will keep the format unchanged.